scripts/jointLklDM.C

```diff
@@ -158,7 +160,9 @@ void jointLklDM(TString configFileName="$GLIKESYS/rcfiles/jointLklDM.rc",Int_t s
   else if(!channel.CompareTo("pi0pi0",TString::kIgnoreCase))     {strchannel = "#pi^{0}#pi^{0}";    minmass=0.135;}
   else if(!channel.CompareTo("gammapi0",TString::kIgnoreCase))   {strchannel = "#pi^{0}#gamma";     minmass=0.135/2.;}
   else if(!channel.CompareTo("pi0gamma",TString::kIgnoreCase))   {strchannel = "#pi^{0}#gamma";     minmass=0.135/2.;}
-  else strchannel = "";
+  else if(!channel.CompareTo("branon",TString::kIgnoreCase))     {strchannel = "branon";            minmass=0;}
+  else                                                           {strchannel = channel;             minmass=0;}
+  TString channellist = "bb_tautau_mumu_WW_gammagamma_pi0pi0_gammapi0_pi0gamma_ee_branon";
```

Comment:
This is not necessary, there should be no special treatment for the single-BR, single-channel case, it should be just a particular case of the general multiple-BR, multiple-channel case

Reply:
Ok!